func/Pipe/DeletePipe.sql

```diff
@@ -0,0 +1,18 @@
+-------------------------------------------------------------------
+-- DELETE PIPE
+-- INPUT:	- v2_model
```

Comment:
Ik zie de meerwaarde van deze functie niet ten opzichte van DELETE FROM v2_pipe WHERE id = x;
Ik zie wel meerwaarde in DeletePipe als die ook automatisch de bijbehorende connection nodes verwijderd als die niet aan iets anders vast zitten